cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceEnsurer.java

```diff
@@ -49,9 +47,7 @@ public Service get() {
         return new AbstractService() {
           @Override
           protected void doStart() {
-            String oldUserId = SecurityRequestContext.getUserId();
             try {
-              SecurityRequestContext.setUserId(Principal.SYSTEM.getName());
               namespaceAdmin.create(NamespaceMeta.DEFAULT);
```

Comment:
The system user, since it creates it. The others will have to be granted manually.